GithubPractice/ViewController.swift

```diff
@@ -48,6 +48,8 @@ class ViewController: UIViewController {
     
     
     @IBAction func didClickClear(_ sender: Any) {
+        defaults.removeObject(forKey: jobKey)
+        defaults.removeObject(forKey: birthDayKey)
     }
```

Comment:
for now is okay. but if you will have more keys to delete, you can remove all the keys in user defaults.

Reply:
oaky, thanks